sysdig/resource_sysdig_secure_benchmark_task_test.go

```diff
@@ -0,0 +1,51 @@
+package sysdig_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/draios/terraform-provider-sysdig/sysdig"
+)
+
+func TestAccSecureBenchmarkTask(t *testing.T) {
+	rText := func() string { return acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum) }
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			if v := os.Getenv("SYSDIG_SECURE_API_TOKEN"); v == "" {
+				t.Fatal("SYSDIG_SECURE_API_TOKEN must be set for acceptance tests")
+			}
+		},
+		ProviderFactories: map[string]func() (*schema.Provider, error){
+			"sysdig": func() (*schema.Provider, error) {
+				return sysdig.Provider(), nil
+			},
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: secureBenchmarkTaskWithName(rText()),
+			},
+			{
+				ResourceName:      "sysdig_secure_benchmark_task.sample",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func secureBenchmarkTaskWithName(name string) string {
+	return fmt.Sprintf(`
+resource "sysdig_secure_benchmark_task" "sample" {
+  name     = "%s"
+  schedule = "0 6 * * *"
```

Comment:
I think the BE will handle the "* * * * *" running each minute, is this true?

Running benchmarks every minute may be a bit expensive for the user, and if a lot of uses run this in a minutely way can be a bit resource exhausting.

Reply:
This is restricted in the backend (to a maximum of once per day), but the code is relatively complex. I think it makes sense to leave the validation in the backend, and let the provider fail if the schedule is too frequent. 

The once per day maximum schedule is noted in the documentation as well.

Reply:
Makes sense, thank you @nkraemer-sysdig :)